src/app/COMPONENTS/task-details/task-details.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core'
+import { Task } from '../../models/task.model'
+import { MaterialModule } from '../../material.module'
+import { TasksService } from '../../service/tasks.service'
+import { ActivatedRoute } from '@angular/router'
+import { Observable, switchMap, take } from 'rxjs'
+import { CommonModule, NgIf } from '@angular/common'
+import { FormBuilder, FormControl, Validators, ReactiveFormsModule, FormGroup } from '@angular/forms'
+
+@Component({
+  selector: 'app-task-details',
+  standalone: true,
+  imports: [
+    MaterialModule,
+    CommonModule,
+    NgIf,
+    ReactiveFormsModule
+  ],
+  templateUrl: './task-details.component.html',
+  styleUrl: './task-details.component.scss'
+})
+export class TaskDetailsComponent implements OnInit {
+  
+  taskForm: FormGroup = this.fb.group({
+    title: new FormControl({ value: "", disabled: true }, Validators.required),
+    description: new FormControl({ value: "", disabled: true }, Validators.required),
+    type: new FormControl({ value: "", disabled: true }, Validators.required),
+    status: new FormControl({ value: "", disabled: true }, Validators.required),
+    createdOn: new FormControl({ value: "", disabled: true }, Validators.required),
+  })
+  
```

Comment:
This is the standard structure when defining variables:
1. inputs
2. outputs
3. observables / behaviorsubj / subjects
4. everything else